### PR TITLE
23/download unzip loop

### PR DESCRIPTION
**Pull request by finish06**
Fixes coderxio/dailymed-api#23

## Explanation
1. `get_zips.py` now calls `extract_zips.py`.  This allows for each SPL to be downloaded, extracted, and deleted before the next SPL is obtained.
2. `get_zips.py` reads and writes the SPL.zip file in chunks to prevent memory issues in low memory environments.
3. `get_zips.py` includes cmd line arguments to specific number of files to unzip, the number of SPLs to download or the specific SPL to download.
```
usage: get_zips.py [-h] [--unzip u] [--download d | --select s]

Download and unzip SPL data.

optional arguments:
  -h, --help    show this help message and exit
  --unzip u     Number of files to unzip from SPL
  --download d  Number of SPL files to download, max 4
  --select s    Specific SPL file to download, i.e. 1, 2, 3 or 4
```

## Rationale
To allow getting zips on the droplet and improve testing in dev environment.

## Tests
1. What testing did you do?
Downloaded SPL files via a variety of cmd line arguments
1. `python get_zips.py`
2. `python get_zips.py --unzip 2 --download 4`
3. `python get_zips.py --unzip 20 --download 1`
4. `python get_zips.py --unzip 2 --select 4`

### CODE REVIEW

File: get_zips.py [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+            type=int,
+            help='Optional number of files to extract from SPL zip.'
+        )
+download_or_select = parser.add_mutually_exclusive_group()
+download_or_select.add_argument(
+                    '--download',
+                    metavar='d',
+                    default='4',
+                    type=int,
+                    help='Optional number of SPL files to download, max 4.'
```
Comment (jrlegrand) [inline get_zips.py:24]: I'm assuming this is the SPL zip files - not individual SPL files - correct?  I didn't understand this initially, so maybe be explicit in this help text.

File: get_zips.py [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+                )
+download_or_select.add_argument(
+                    '--select',
+                    metavar='s',
+                    type=int,
+                    help="Optional SPL file to download, i.e. 1, 2, 3 or 4"
```
Comment (jrlegrand) [inline get_zips.py:30]: (ditto of above) I'm assuming this is the SPL zip files - not individual SPL files - correct?  I didn't understand this initially, so maybe be explicit in this help text.

File: get_zips.py (hunk 1 above)
Comment (jrlegrand) [reply get_zips.py:24]: I wonder if there's a way to programmatically know the max number of zip files from DailyMed.  Not an issue now, but I would imagine there will eventually be 5 zips, etc.

**Review by jrlegrand — APPROVED** [review]

File: get_zips.py (hunk 2 above)
Comment (finish06) [reply get_zips.py]: Updated to spl zips.

File: get_zips.py (hunk 1 above)
Comment (finish06) [reply get_zips.py]: Updated to spl zips.

**Comment by finish06** [comment]
Reran the following scenarios with success:
```
python get_zips.py
python get_zips.py --unzip 2 --download 4
python get_zips.py --unzip 20 --download 1
python get_zips.py --unzip 2 --select 4
```

**Comment by finish06** [comment]
Help menu:
```
usage: get_zips.py [-h] [--unzip u] [--download d | --select s]

Download and unzip SPL data.

optional arguments:
  -h, --help    show this help message and exit
  --unzip u     Optional number of files to extract from SPL zip.
  --download d  Optional number of SPL zip files to download, max 4.
  --select s    Optional SPL zip file to download, i.e. 1, 2, 3 or 4
```

**Pull request merged**